proto/regen/ecocredit/v1/tx.proto

```diff
@@ -15,16 +15,15 @@ option go_package = "github.com/regen-network/regen-ledger/x/ecocredit/base/type
 service Msg {
 
   // CreateClass creates a new credit class under the given credit type with an
-  // approved list of issuers and optional metadata. The fee denom must be one
-  // of the denoms listed in Params.credit_class_fee and greater than or equal
-  // to the fee amount but only the minimum amount is charged. The creator of
-  // the credit class becomes the admin of the credit class upon creation.
+  // approved list of issuers and optional metadata. If the class fee parameter
+  // is set, the fee field must be populated. The creator of the credit class
+  // becomes the admin of the credit class upon creation.
```

Comment:
Any reason for removing the requirements around the fee amount? The fee denom must match and the amount must be grater that or equal to the fee amount but only the minimal amount is charged.

Reply:
the wording seemed related to when we had multiple fees in params. was the "minimal amount" intended to be related to the minimum fee listed in params? or was that to describe the scenario where they provided say "30regen" when the fee was "20regen"?

Reply:
The latter, which still seems relevant to me.

Reply:
https://github.com/regen-network/regen-ledger/pull/1556/commits/c14b7de9e02ce023e2dd3ce0f9737f0fc7c7114c